ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml

```diff
@@ -85,28 +85,25 @@ tests:
   commands: commitchecker --start ${PULL_BASE_SHA:-main}
   container:
     from: commitchecker
+- as: check-sync-pr-valid
```

Comment:
nit: "valid" is not the clearest 
"no-orphaned-carries-on-sync" ?

Reply:
I was keeping the same name...